sbol_utilities/sbol3_genbank_conversion.py

```diff
@@ -58,7 +62,7 @@ class GenBank_SBOL3_Converter:
     gb2so_map = {}
     so2gb_map = {}
     default_SO_ontology = "SO:0000110"
```

Comment:
Make an issue to consider whether this should actually be "SO:0000001" (disagreeing with libSBOLj)

Reply:
Done, created [here](https://github.com/SynBioDex/SBOL-utilities/issues/146).